packages/react/src/components/Details/Details.stories.tsx

```diff
@@ -85,6 +90,25 @@ export const InCardWithColor: StoryFn<typeof Details> = () => (
   </>
 );
 
+export const MultipleChildren: StoryFn<typeof Details> = () => (
+  <u-details class='ds-details'>
+    <u-summary>Vi har følgende informasjon om deg</u-summary>
+    <p>
+      For å kunne tilby bedre tjenester til våre innbyggere henter vi ut
+      følgende informasjon fra Folkeregisteret, matrikkelen og kontakt- og
+      reservasjonsregisteret:{' '}
+    </p>
+    <List.Unordered>
+      <List.Item>
+        <PersonIcon aria-label='' fontSize='1.5em' /> Navn <p>Ken Langaas</p>
+      </List.Item>
+      <List.Item>
+        <PhoneIcon fontSize='1.5em' /> Telefon <p>999 99 999</p>
+      </List.Item>
+    </List.Unordered>
+  </u-details>
+);
+
 export const Controlled: StoryFn<typeof Details> = () => {
```

Comment:
This story does not add any value to our storybook, I suggest removing it

Reply:
I have not work much with the testing, but could it be moved there? Don't want this fix to reoccur.